pr-statistics-spent-time.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+
+
+# PARAMETERS ----------------------------
+GITHUB_REPO="status-im/status-dev-cli"
+DEBUG=1
+
+# PLATFORM ----------------------------
+case "$OSTYPE" in
+  darwin*)  OS="DARWIN" ;;
+  *)        OS="LINUX" ;;
+esac
+
+# FUNCTIONS ----------------------------
+
+github_pr() {
+  repo="$1"
+  list_spent_time=""
+  list_dt_closed=""
+  list_dt_created=""
+
+	last_page=$(curl -i -sL "https://api.github.com/repos/$repo/pulls?state=closed" | grep rel=\"last\" | cut -d "," -f 2 | cut -d "=" -f 3 | cut -d ">" -f 1)
+  [ "$last_page" = "" ] && last_page=1
+  [ "$DEBUG" = "1" ] && echo "DEBUG -- url is https://api.github.com/repos/$repo/pulls?state=closed -- nb page : $last_page" >&2
+  for i in $(seq 1 $last_page); do
```

Comment:
Can we print a progress bar? like

[Fetching PR lists: 1/75]
[Fetching PR lists: 2/75]

etc.

Also, are there any timeout/GH API block concerns?

Reply:
done in commit https://github.com/status-im/status-react/pull/2893/commits/0b287d778877586a7249552f78af30f2223027ac

There is no timeout concerns, but github api allow us to get PR by batch of 30 max only